### ISSUE TITLE
CoC report received 

### ISSUE THREAD

**Issue by hackygolucky** [nodejs/TSC#324]
A Code of Conduct violation report was received via report@nodejs.org and published publically [here](https://www.reddit.com/r/node/comments/6whs2e/multiple_coc_violations_by_nodejs_board_member/). We take this report seriously and both the @nodejs/tsc and @nodejs/community-committee must follow the [Moderation Guidelines](https://github.com/nodejs/TSC/blob/master/Moderation-Policy.md#moderation-policy) and documented processes to proceed in the private moderation repo. Because this is also a Director, the Board of Directors for the Node.js Foundation has also received the complaint.

If you have any questions, please talk with the @jasnell @mhdawson @williamkapke @hackygolucky about procedures moving forward.

**Comment by RobertWHurst** [comment]
Here is an __archive.is snapshot__ of the [reddit thread](http://archive.is/AeYJf) in case the original source is removed.

**Comment by benjamingr** [comment]
Is this a good place to discuss it publicly in the project?

After reading the complaint and relevant tweets - I don't think any of the issues in this report constitutes of a CoC violation at all. I think we should take this complaint seriously - but I don't see anything actionable here. It got a lot of upvotes though - Am I missing anything obvious? 

I'm sad looking at things like the following reported as a CoC violation and inclusivity barrier:

> "never underestimate the wrath of a mildly inconvenienced white dude (and yes it is all dudes complaining)"

In my opinion it illustrates people don't really understand the challenges we face as a community and disadvantaged segments of the population in particular face. I think this is a communication problem and I'll open an issue in CommComm to discuss it in particular.

**Comment by joepie91** [comment]
@benjamingr I think the question to address here is whether the purpose of the CoC is to create a maximally inclusive environment and equal treatment *in general*, or whether its purpose is to advantage 'disadvantaged groups' (for lack of a better term).

In the former context, the quoted remark would absolutely be a CoC violation (racist/sexist remarks that reduce inclusivity unnecessarily), but in the latter it wouldn't be as, within anglophone culture, "white men" are generally not considered a disadvantaged group.

However, if the purpose *is* the latter, I would expect that the CoC would be much less well-received by the Node.js userbase at large (myself included).

I think there are probably some cultural differences to hash out here, as well - the current approach to 'inclusivity' seems very oriented around anglophone culture (and US culture in particular), and this introduces a lot of hard-to-understand polarization from the perspective of non-US users.

**Comment by jasnell** [comment]
After reviewing this yesterday and doing some investigation on the Reddit threads (and being reminded yet again why I never actually use Reddit for anything), my recommend is for the TSC to disregard this report as it appears to be part of a targeted harassment campaign against the reported individual.

While the comments made by the individual on twitter are indeed distasteful at times (even as "jokes"), the individual is not and has not been an active contributor to core, and is not an active participant in discussions around core. The reported comments all occurred on Twitter and not in any TSC managed repositories.

The individual does have interactions with both the Community Committee and Foundation Board, so I believe both of those committees will need to look at the report separately on their own to make their own decisions on whether and how to proceed.

As a TSC issue, however, I think we can close this.

**Comment by benjamingr** [comment]
Hey @joepie91, thanks for explaining your perspective. As someone who does not live in US culture and whose (white, originally German/Polish) family has been subject to racism in the past - I'm very aware of the problem of being "US culture specific". I'm not saying I agree with everything (or anything for that matter) Ashley wrote, only that I do not feel that it is a CoC violation.

I don't think this is about promoting disadvantaged groups at all. Do you not see a difference between the two statements below?

 - "White men often make poor programmers".
 - "Black women often make poor programmers".

Personally, from my non-US perspective (and do feel free to correct me here, I don't live in the US and don't know the political atmosphere):

 - The first one will not be taken seriously by anyone - the majority of the Node.js community and the programming community as a whole are white men. As a white man, I don't feel excluded when I see it. I might be wrong here, and there might be people who genuinely take offense at this (at which case, please inform me).
 - The second one is a lot more exclusive since it singles out a minority group in the project. That group is a lot less represented in the project anyway and a member is a lot more likely to take offense. This doesn't have to be about disadvantaged group necessarily - if it was "all people who live in Houston, Texas are bad programmers" I would still see it as exclusive.

I understand this is my _personal_ point of view and how I'm reading this. The way I'm seeing it the CoC isn't about "fixing the world" - it's about making the informed decision to not let gender, religion, ethnicity, race or anything similar to _be a factor_ so that people can contribute.

I don't have a stake here, I don't know Ashley from before and I'm not a CommComm member. I have not been involved in the issue and am not a CTC or TSC member - I and am only a collaborator discussing what is visible to me and trying to understand why the report was made.

If a group of people want to promote a certain minority (LGBT programmers for instance) then I think it's a positive initiative that we as a community should embrace - especially since it is not coming at the expense of anything. Inclusivity isn't about doing those initiatives - it's about enabling them in the first place so that Node.js isn't a locked garden like it used to be.

I think working towards inclusivity has been a really positive factor in the project. I think the project is generally doing really well and that a lot of really nice features have landed in the past year - a lot of those features are by people who are not your "standard" open source collaborators.

**Comment by benjamingr** [comment]
@jasnell the discussion is about the particular report. If there is anywhere (in the org) that is more appropriate for the discussion I'll gladly move it. So far I'm learning from the discussion with @joepie91 so if it doesn't bother you I'd like to keep it.

I agree about the TSC or CTC not having an action item here.

**Comment by jasnell** [comment]
@benjamingr .. ok, no problem then :-) feel free to continue!

**Comment by mikl** [comment]
The code of conduct does not (and should not) grant a free pass that allows negative behaviour towards majority groups.

If you think that disparaging a group, or spreading harmful stereotypes about it, should be disallowed, then this needs to be applied universally.

Why, you might ask? Because universal principles are the foundation for justice. Selective enforcement breeds resentment, as we know well from examples in history.

The only way to achieve mutual respect between groups, is for the groups to behave respectfully towards each other. If group Y is constantly berated for their disrespect towards group X, but allow group X to disrespect group Y as much as they please, group Y will come to resent the rules, the system and group X.

In short, if you are perceived to establish the precedent that rules only protects certain groups, you can’t expect the unprotected groups to respect the rules.

**Comment by joepie91** [comment]
@benjamingr I would personally say that neither of those two statements is acceptable or desirable. Even more so because there's no *reason* for either statement to be made; there's nothing to be gained from them as a community, they can only really be antagonistic in nature.

I have personally spoken to a number of people who, throughout the past several years, have been made to feel unwelcome in the Node.js community because of sexist/racist/otherwise-discriminatory remarks that served no constructive purpose.

A recurring theme was that they felt their complaints about this weren't or wouldn't be taken seriously, because they weren't part of a 'disadvantaged group' - or not a part of the *'right'* one, for the remark in question. Some expressed concerns of retaliation from "SJWs" (their words) because of this.

I personally don't think that this kind of polarization should have a place in the community, regardless of which 'side' is speaking about which other 'side'. I also don't think it's the right choice to try and guess at whether people will feel excluded by particular remarks; everybody has their own unique set of personal experiences and worldviews, and their interpretation of a remark may differ *wildly* from that of somebody else in the same nominative 'group'.

Rather than trying to assess whether there might be specific people who feel unwelcome because of them - which is difficult to do accurately, because often people will not speak out about it publicly - I think the better solution is to just consider *any* discriminatory remark undesirable under the CoC. If discriminatory remarks are considered unacceptable in general, I don't think there's a need to distinguish between 'groups' in the first place.

As an aside; I've noticed that the polarization (on both sides) originates from a fairly limited group of actors, and that the broader userbase either doesn't care and ignores it, or gets increasingly irritated by it and tries to avoid the venues where the resulting drama plays out. I don't intend to name these groups publicly here, as this is likely to attract trolls, but I would be happy to discuss this privately.

> The way I'm seeing it the CoC isn't about "fixing the world" - it's about making the informed decision to not let gender, religion, ethnicity, race or anything similar to be a factor so that people can contribute.

I feel that this is the preferable (and most constructive) interpretation, but I also think that this needs to be enforced as such - that is, CoC violations should be taken seriously regardless of exactly who the intended target is, and without trying to guess at what the motives of the reporter are; assuming that there is sufficient evidence, of course.

For example, I understand that this particular instance is not one for the TSC to handle due to non-involvement, but I think that dismissing the report on the basis of "it's part of a targeted harassment campaign against the reported individual" is the wrong thing to do; the report contains sourced statements that violate the CoC (if not in letter, then in spirit) and these speak for themselves.

**Comment by jasnell** [comment]
@joepie91 I certainly cannot disagree with any part of that. The whole conversation around CoCs is so charged with vitriol on every side that those of us who do try to enforce them as fairly and objectively as possible end up getting slammed and run through the mud consistently by one side or the other. It makes it difficult, but no less the right thing to do.

**Comment by benjamingr** [comment]
@joepie91 

>  I would personally say that neither of those two statements is acceptable or desirable. 

I agree, and I don't think it's particularly helpful in healing. I still I don't see how saying things like "never underestimate the wrath of a mildly inconvenienced white dude" consists of a CoC violation. I see a huge difference between that and a remark that has the potential to exclude members or make them feel unwelcome.

I think there is a big difference between targeting 92% of all programmers and targeting 8%.

> I have personally spoken to a number of people who, throughout the past several years, have been made to feel unwelcome in the Node.js community because of sexist/racist/otherwise-discriminatory remarks that served no constructive purpose.

To be fair I've seen tremendous improvement over the last few years. All the feedback I've received when on-boarding contributors or discussing the community has been overwhelmingly positive - minority groups included. I acknowledge how much progress has been made in Node.js in the last few years under the current leadership in that regard. 

When I joined io.js after the split a lot of the reason why is because I felt I could make a difference where in Node.js I felt like I was being ignored. (I even [spoke about it](https://www.youtube.com/watch?v=LGpmUyFnyuQ)). I think Node.js got _a lot better_ in that.

If you do talk to people who feel discriminated against - please do bring that up. 

>  I don't intend to name these groups publicly here, as this is likely to attract trolls, but I would be happy to discuss this privately.

My email is on the front page https://github.com/nodejs/node - if you'd prefer to discuss this privately I'll happily oblige.

----  

 > > The way I'm seeing it the CoC isn't about "fixing the world" - it's about making the informed decision to not let gender, religion, ethnicity, race or anything similar to be a factor so that people can contribute.

> I feel that this is the preferable (and most constructive) interpretation, but I also think that this needs to be enforced as such - that is, CoC violations should be taken seriously regardless of exactly who the intended target is, and without trying to guess at what the motives of the reporter are; assuming that there is sufficient evidence, of course.

I agree with that 100%. I am not calling this an attack (although the timing raises questions) and I don't think the motives should be second guessed - but I still don't see where the CoC is violated. I acknowledge some people might find other things I don't find as offensive - offensive and I guess we'll wait until the issue has been discussed in CommComm and see what the outcome is.

**Comment by mcollina** [comment]
At this point our CoC does not tolerate harassment in any form. Even if that harassment is towards a member of the so-called "majority group". IMHO there is no difference who is the target.

**Comment by benjamingr** [comment]
@mikl 

> If you think that disparaging a group, or spreading harmful stereotypes about it, should be disallowed, then this needs to be applied universally.

> In short, if you are perceived to establish the precedent that rules only protects certain groups, you can’t expect the unprotected groups to respect the rules.

First and foremost I think [Node.js hasn't done a great job](https://github.com/nodejs/community-committee/issues/117#issuecomment-325637045) explaining what the CoC is for or engaging with these community members.

Still, I think that context here makes a huge difference. You can't really "single out" a majority but it's very easy to accidentally single out a small minority.

The part I disagree here with is that Ashley is spreading "harmful stereotypes" in her tweets here. I hope it's obvious I'm not saying Node.js should discriminate people based on them being in a majority group - had someone targeted a specific individual that just happened to be a part of that group I would feel very different.

Also note that I am _explicitly not_ voicing an opinion about what happened with Rod. I am not very happy with the way that was handled (not sure that in the way you'd expect).


----

There are a number of silent people reading this issue and 👍  👎 ing comments. Please do speak up if any of you feel like Ashley's comments have excluded you or made you feel unwelcome in the project. I honestly don't see why "never underestimate the wrath of a mildly inconvenienced white dude" would exclude "white dude"s from the project.

**Comment by joepie91** [comment]
> I still I don't see how saying things like "never underestimate the wrath of a mildly inconvenienced white dude" consists of a CoC violation. I see a huge difference between that and a remark that has the potential to exclude members or make them feel unwelcome.

This *has* such a potential, though. This very clearly gives off a "white men are not welcome and/or their concerns will be trivialized" vibe to many people. I can't speak for this particular remark, but this *type* of remark certainly has played a role in the 'people feeling unwelcome' that I've mentioned before.

Just like constant subtle insinuations about women being somehow inferior produces a toxic and unwelcoming environment for many women, this kind of remark does the same for many white men; even if there's no obvious 'smoking gun'. One group of people is no less emotionally affected by remarks like this than another group, especially if they are of an ongoing nature. It creates the appearance of an 'ingroup' or 'clique'.

> I think there is a big difference between targeting 92% of all programmers and targeting 8%.

I don't think the numbers matter here at all, really. The interpretation of social interactions is something that happens on a largely individual basis; somebody is going to be either offended or not offended by a remark, largely regardless of whether the arbitrary 'group' they've been classified as is the majority in any given situation or not. Idem for feeling welcome or unwelcome.

In the end, people experience things on a personal basis, not as a homogeneous group; especially on a medium like the internet that doesn't involve a physical presence. It's just from the *outside* that it looks harmless, if you're not personally affected by it. This is why I always make an effort to speak to people personally in conflicts, even if I disagree with their views or claims.

> To be fair I've seen tremendous improvement over the last few years. [...] If you do talk to people who feel discriminated against - please do bring that up.

I do agree that Node.js has been doing reasonably well *overall*, but there definitely are still some issues to fix; in particular those that aren't often mentioned (for eg. the fear-of-retaliation reasons I've described before). Nevertheless, I don't intend to make light of the work that has already been done in this area :)

Unfortunately I cannot easily get in contact with those I've spoken to in the past, and their issues were often not specifically about the TSC but rather about various other aspects of the community; but I hope that this thread can serve as encouragement for people to report this kind of thing in the future, if/when it reoccurs.

> My email is on the front page https://github.com/nodejs/node - if you'd prefer to discuss this privately I'll happily oblige.

I'm going to have to step out of this thread/topic for a while for work reasons, but I'll certainly shoot you an e-mail somewhere later this week. Thanks!

**Comment by joshmanders** [comment]
> There are a number of silent people reading this issue and 👍 👎 ing comments. Please do speak up if any of you feel like Ashley's comments have excluded you or made you feel unwelcome in the project. I honestly don't see why "never underestimate the wrath of a mildly inconvenienced white dude" would exclude "white dude"s from the project.

For what it's worth, I have been harassed by Ashley and other prominent members of the Node.js community in the form of belittling and even gone as far as not outright calling me a rapist, but comparing me to one, because I replied to a tweet.

I've even reported the incidents and after that happened, a moderation thread was created in my name and I was kicked out of the node.js organization as a paying member, and blocked from even being told what I had done that violated it.

Those events has caused me to not care to contribute to the project anymore.

**Comment by azah** [comment]
Let me preface this by saying I'm only speaking from my perspective.

I don't think really anyone, myself included, is excluded by singular statements like
"never underestimate the wrath of a mildly inconvenienced white dude".

I do, however, find it just distasteful. I was raised to not make stereotypical statements about anyone and especially not a whole group. I find it troublesome that it's permissible for a member of a community to write such things when I would get vigorously lambasted for daring to write something remotely similar. I would never extrapolate about an entire sex or race based on a few people; yet tweets like this are permissible because it's towards a majority? Whatever happened to two wrongs never make a right?

More importantly, I get the sense that such a Code of Conduct will be applied *much* more harshly towards me than a minority, regardless of the allegations. 

I do feel discouraged from participating in this community. I get the sense that my concerns will not be taken seriously because my skin color and sex; attributes I have no control over, didn't ask for, and didn't get to choose. 

I also feel discouraged from participating in discussions because I feel like if I ever say something that is even slightly critical, I may get witchhunted for it. So I don't talk that much.

What I'm saying is

If there is a Code of Conduct, it must be applied evenly to members within the community. (I'm not talking about this tweet in particular. Just in General) Anything else speaks volumes about this entity's leadership. Which is why I think people are upset: This is just an indicator of a larger problem.

**Comment by refack** [comment]
IMHO @jasnell's [comment](https://github.com/nodejs/TSC/issues/324#issuecomment-325657295) on deferring the decision to the Community-Committee and the Board makes a lot of sense. The reported individual has minimal interaction with the Technical side of the foundation (in TSC, the T is for Technical), hence this should be considered in the forums they are active in.

**Comment by mikl** [comment]
@benjamingr 

> You can't really "single out" a majority but it's very easy to accidentally single out a small minority.

Straw man. No one talked about singling out anyone.

> I honestly don't see why "never underestimate the wrath of a mildly inconvenienced white dude" would exclude "white dude"s from the project.

Another straw man. A statement doesn’t have to be strong enough to “exclude” anyone (if that was even possible), to be in violation of the rules.

I guess you think of that as some sort of joke, but it still projects a stereotype that “white dudes” gets upset over small things, presumably because we have never known hardship.
Also, if you read the whole complaint, there’s also a lot more where it came from, including the infamous “KILL ALL MEN”, and “FUCK YOU MEN”.

**Comment by tkambler** [comment]
@jasnell -

Whenever these discussions occur, the dregs of society inevitably seep their way into the conversation. (By "dregs," I'm referring to anyone who uses nasty, hateful language that targets people based on their gender / orientation / religion / etc...). This entire controversy had its roots in some thoughtful discussions, but those have gone by the wayside. It has turned into a black / white, us / them issue for which there can be no middle ground for a lot of people.

Personally, here's where I'm at:

Given the events of the past several days, I think the need for a clear / concise / actionable CoC is plainly clear. I don't argue that point - I'm with you. My concern arises from the fact that so many of the crucial points that these CoCs outline are flagrantly broken by members of the Node / npm boards on an on-going basis. _That's_ what this is about. That's it - nothing else!

Before I continue - can we come to an agreement on one point (please)? At a bare minimum, can we come to an agreement that some of these individuals do in fact break these rules on an on-going basis? (At the moment, I'm specifically referring to Kat Marchán and Ashley Williams.) If you want specific examples, I can provide them, but I think that horse has already been beaten to death. Furthermore - If you want to debate the point as to whether their Twitter feeds constitute npm-centric communications, we can have that talk. But I'm not even referring to that right now. If you set that aside, and just consider the messages in and of themselves - do they break the rules that you have laid down? I believe the clear answer to that question is a solid _yes_ - they do.

Now, if we can at least agree on _that_ point - let's talk about whether their Twitter feeds constitute messages that could be considered to be npm-centric communications. These individuals prominently list themselves as working for the npm organization and engage with members of the communication through these channels. I believe that to be an tacit endorsement by npm for what they say.

May I make a suggestion? These individuals should:

1.) Remove references to npm from their Twitter feeds and stop engaging with members of the community using these platforms.

OR -

2.) These individuals should create separate Twitter accounts to be use specifically for npm-related matters. Any snide remarks, etc..., that they want to make could then be continued to be made using their personal accounts.

Am I being unreasonable here?

**Comment by jasnell** [comment]
> At this point of CoC does not tolerate harassment in any form.

I think this is really the key part. Harassment is harassment, no matter who it is directed at, and we absolutely *must* be fair and objective in how we enforce the CoC.

However, we must also be aware that there are individuals in the community who feel that harassment of any form against {Insert Particular Group Name Here} is fully justified because {Insert Particular Group Name Here} somehow "deserves" it. Playing along with such harassment `!==` Code of Conduct enforcement. We cannot let ourselves be sucked into that kind of game playing. We *do* have an obligation to take *every* report seriously unless it can be shown that the report is nothing more than a form of harassment. Even then, we still have an obligation to consider the report.

Unfortunately, this can end up being quite a subjective decision to make, so we need to look at more objective measures. In this particular case, the individual in question (a Foundation Board member and a CommComm member) is *not* a contributor to core. The TSC *can* choose to limit the individuals ability to participate in TSC-managed repositories by removing their write access to those but that is the extent of the action the TSC can take. We cannot ban the individual from all participation because the TSC does not have oversight over either the Board or the CommComm, and their participation in both grants the individual a certain level of access. That is a purely objective measure.

I will say that this report is being looked at by both the CommComm and the Foundation Board, independently of the TSC, and it _is_ being taken seriously. Aside from any targeted harassment campaign that may be occurring, this is not the first time we have received feedback from people in the community expressing concerns about the individual in question, I just believe that given the lack of active participation on the technical side of things, the TSC is not the correct venue for handling this particular case.

My plan at this point is to at least raise this issue for consideration by the TSC/CTC during the private section of the upcoming meeting, with my personal recommendation that the issue should be deferred to the other committees. But I want to give the full TSC/CTC the opportunity to weigh in.

**Comment by ajhyndman** [comment]
I am not an invested party in this conversation, and if my comments are unwelcome, I am happy to retract them.

I want to make sure that we do not make the mistake in this issue of equating the way stereotypes or other forms of antagnosism affect a disadvantaged group, versus those which target an established majority.  While, of course, malice or harrassment in any form is never welcome in a community, that is also not in question here, and there is a danger in equating them.  Actions directed toward a majority have a disproportionate effect to actions directed toward a minority, and to ignore that is to ignore a very common form of oppression.

Being ridiculed or mocked for being a member of a majority group may result in loss of face in argument, or blocked by someone on twitter.  Being ridiculed or mocked for being a member of a disadvantaged group (or worse, defending yourself as a disadvantaged group) much more commonly result in things like job termination, dismissal in interviews, misattribution of achievements to others around them, and the list goes on.

**Comment by ajhyndman** [comment]
Once you have accepted the premise that "Harassment is harassment", whether directed at an individual or the dominant group in a community, you have okayed systematic oppression of minorities.

**Comment by jasnell** [comment]
@tkambler 

> Before I continue - can we come to an agreement on one point (please)? At a bare minimum, can we come to an agreement that some of these individuals do in fact break these rules on an on-going basis?

I can certainly agree that had *any* collaborator made these kinds of statements within the Node.js GitHub tracker they *would* be considered to be flagrant and obvious Code-of-Conduct violations.

**Comment by squarefractal** [comment]
@benjamingr 

> Please do speak up if any of you feel like Ashley's comments have excluded you or made you feel unwelcome in the project.

As another person whose incidentally has the sex of the "majority group", those comments are indeed do make me feel ostracized from the nodejs community.

> After reviewing this yesterday and doing some investigation on the Reddit threads [...], my recommend is for the TSC to disregard this report as it appears to be part of a targeted harassment campaign.

I would kindly request the administration to evaluate this report objectively, regardless of what certain members on Reddit say. Not all parts of Reddit are welcoming and I understand that, but the report itself has merit.

**Comment by benjamingr** [comment]
@mikl I ask that you reconsider your previous comment and offer something that is more in the line of discussion instead of calling my arguments a "straw man". The context of this discussion is [just a few comments above](https://github.com/nodejs/TSC/issues/324#issuecomment-325659277).

**Comment by jasnell** [comment]
@squarefractal and others, just to reiterate my comment [here](https://github.com/nodejs/TSC/issues/324#issuecomment-325693240):

> My plan at this point is to at least raise this issue for consideration by the TSC/CTC during the private section of the upcoming meeting, with my personal recommendation that the issue should be deferred to the other committees. But I want to give the full TSC/CTC the opportunity to weigh in.

And, you have my assurances that I personally look at this report with the same objectivity that I look at all such reports (fortunately, we really do not receive that many)

**Comment by rachelnicole** [comment]
I agree it's time to close & move it to the board & commcomm

**Comment by ilovednodejs** [comment]
I'm having a hard time understanding how it is that some of you do not find this offensive. I do.
This should be an extremely easy problem to solve. Would you feel comfortable saying the same text lines she tweeted?

I know I would not.
I would never dare to say such things.

I would also like to add

https://archive.is/6fJ6E (why?)
https://archive.is/WHxJG (why does she hate the PHP creator?)
https://archive.is/679QC In another context this would have caused an uproar
https://archive.is/kGKZD Same

Please consider the CoC violation seriously

**Comment by jakeNiemiec** [comment]
>Please do speak up if any of you feel like Ashley's comments have excluded you or made you feel unwelcome in the project.

After what happened to Rod (the twitter part), I feel like there needs to be a clarification:

- should a personal twitter account be subject to CoC?

IMHO: let Twitter be a private space (for all parties).

**Comment by Fishrock123** [comment]
As this was originally intended as a notice only here, I am going to lock and close this.

Please create new issues for specifics if you feel so inclined.

**Comment by jasnell** [comment]
Please do not close this yet, I plan to raise the issue for TSC discussion during the private portion of the upcoming meeting.

**Comment by mcollina** [comment]
Closing the issue for now.
